### DISCUSSION TITLE
alist客户端挂载不上115网盘，怎么回事？

### DISCUSSION THREAD

**Discussion by Dryangxiansheng** [AlistGo/alist#6971] [Q&A]
failed to mount FUSE fs: mount stopped before calling Init: mount failed: cgofuse: cannot find winfsp
mount finished with code 1 and signal null
![图片](https://github.com/user-attachments/assets/3d5e4aba-c048-4900-a985-2e924a585e51)
![图片](https://github.com/user-attachments/assets/90ea0aaf-e48a-43cb-9895-4a66b2ac9547)

**Comment by anwen-anyi** [comment]
这个安装了吗  试试
![image](https://github.com/user-attachments/assets/550242eb-3abe-4557-a2b9-afebed1a70ad)
